avatar.css

```diff
@@ -0,0 +1,24 @@
+.avatar-container{
+    display: flex;
+    flex-direction: row;
```

Comment:
I would also suggest you to add `flex-wrap: wrap` to `.avatar-container` as you might face issues while trying to develop for mobile device widths.